src/main/java/io/minepkg/testutils/TestUtils.java

```diff
@@ -15,15 +15,19 @@
 import net.minecraft.util.registry.Registry;
 import net.minecraft.world.GameRules;
 import net.minecraft.world.GameRules.BooleanRule;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class TestUtils implements ModInitializer {
+  public static final String MOD_ID = "testutils"; // Note: currently different from fabric.mod.json id :/
```

Comment:
Note: Note the note.

Ideally, both ids should be the same.
The only effect of changing `MOD_ID` would be that old rulebooks cannot be loaded.
Since this is a creative tool either way, I'd suggest changing the `MOD_ID` to `"test-utils"`.

Reply:
should be fine but we need to update https://github.com/minepkg/test-mansion then as it includes the item in the world.